src/Sylius/Bundle/OrderBundle/Controller/OrderController.php

```diff
@@ -16,7 +16,7 @@
 use FOS\RestBundle\View\View;
 use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
 use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
-use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Order\Repository\OrderRepositoryInterface;
```

Comment:
This makes the call in `summaryAction` incorrect - `$cart = $this->getOrderRepository()->findCartForSummary($cart->getId());`.

The solution which might work:
 * replace `findCartForSummary` with `findCartById` here
 * copy the existing method implementation to the `OrderController` in `CoreBundle`

Reply:
I made the modifications you recommended to me

Reply:
@pamil is it ok for you ? thx